vidcutter/libs/videoservice.py

```diff
@@ -511,7 +523,11 @@ def getKeyframes(self, source: str, formatted_time: bool=False) -> list:
                     keyframe_times.append(timecode[:-3])
                 else:
                     keyframe_times.append(float(timecode))
-        last_keyframe = self.duration().toString('h:mm:ss.zzz')
+        if formatted_time:
+            last_keyframe = self.duration().toString('h:mm:ss.zzz')
+        else:
+            last_keyframe = self.parent.qtime2delta(self.duration())
```

Comment:
I have to say, `qtime2delta()` — which I realize you didn't write, of course — strikes me as kind of pointless. Not just here, but everywhere.

It doesn't even _return_ an actual timedelta... it just creates one, stuffs some values into it, and then converts it to a microsecond-resolution float. (Even though the data is only at millisecond resolution, really.)

QTime has `QTime::msecSinceStartOfDay` that can accomplish exactly the same thing, if the result is divided by 1000.

I mean, heck, `delta2QTime` basically does exactly that in reverse:
https://github.com/ozmartian/vidcutter/blob/154e0e11c43e91cb7c299f86377da8d51f77a7c7/vidcutter/videocutter.py#L1313-L1318